README.md

```diff
@@ -17,3 +17,28 @@ How to start [testing](https://github.com/teksi/wastewater/discussions/72)
 Upcoming first version TEKSI Wastewater 2024.0 is due to end of march 2024.
 
 Migration path from QGEP to TEKSI Wastewater is due to end of summer 2024. <!---  // skip-keyword-check -->
+
+## Local development
+
+1. Open OSGeo4W Shell and run `pip install debugpy`.
+2. Clone this repository to your local machine.
+3. Open QGIS, go to the _Environment_ section in `Settings > Options - System` and add the following custom variables:
+
+   | Apply  | Variable                 | Value                   |
+   | :----- | :----------------------- | :---------------------- |
+   | APPEND | QGIS_PLUGINPATH          | {repositoryPath}/plugin |
+   | APPEND | QGIS_PLUGIN_USE_DEBUGGER | debugpy                 |
+
+4. Install QGIS plugin _Plugin Reloader_. This will allow you to reload the plugin without restarting QGIS.
```

Comment:
This is not working at the moment if the project is already open. To be able to use the plugin reloader we need to check on load if the wastewater project is open, and in that case pass the database configuration to the DatabaseUtils class https://github.com/teksi/wastewater/blob/main/plugin/teksi_wastewater/teksi_wastewater_plugin.py#L614

Reply:
Q: What is your proposal? 

- Add the capability to use the Plugin Reloader by checking on load if the project is open.
- Remove this comment as QGIS with the project can easily be started by VS Code.

Reply:
you can leave the comment, I added a fix for that here: #434

Reply:
Fix merged.